l10n_it_vat_registries/report/report_registro_iva.xml

```diff
@@ -142,10 +142,18 @@
                                                             /></td>
                                                 </t>
                                                 <!-- Ragione sociale -->
-                                                <td class="left_without_line_bold"><div
+                                                <td class="left_without_line_bold">
+                                                    <div
+                                                            t-if="not show_full_contact_addess"
                                                             style="page-break-inside: avoid"
                                                             t-esc="move.partner_id.name"
-                                                        /></td>
+                                                        />
+                                                    <div
+                                                            t-if="show_full_contact_addess"
+                                                            style="page-break-inside: avoid"
+                                                            t-esc="move.partner_id.contact_address"
```

Comment:
non è più corretto in questo modo?

```
t-field="move.partner_id" 
t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'
```

che ne pensi?

Reply:
Se con `contact_address` si ottiene lo stesso risultato con meno righe, come mai vedi più corretto `t-field`?

Reply:
perché si usa in tutti i report e nel caso di inherit se qualcuno vuole impostare la `partita iva` e facile basta cambiare l'attributo `t-options`

Reply:
@Borruso grazie, modificato

Reply:
@Borruso come non detto, ho di nuovo modificato, per 2 motivi: non andare a capo su troppe righe e permettere una migliore ereditabilità, vedi https://github.com/OCA/l10n-italy/pull/4023